docs/source/best-practices/clusterusage.md

```diff
@@ -27,3 +27,60 @@ If you are attempting to transfer data, we have a dedicated transfer node that y
 :::{seealso}
 {ref}`transferring-data`.
 :::
+
```

Comment:
I'm currently in the process of rewriting this. This section is inaccurate given that historical-seff did not exist on explorer for people to use until ~ an hour ago